TYPO3.Flow/Classes/TYPO3/Flow/Http/RequestHandler.php

```diff
@@ -111,8 +116,8 @@ public function handleRequest()
             $this->request->setBaseUri(new Uri($this->settings['http']['baseUri']));
         }
 
-        $componentContext = new ComponentContext($this->request, $this->response);
-        $this->baseComponentChain->handle($componentContext);
+        $this->componentContext = new ComponentContext($this->request, $this->response);
```

Comment:
Yep, ok, so we could only initialize the component context pre boot, that will work. We should declare it as Flow\Proxy false though and comment that it is created pre boot.